evm_loader/test_transaction.py

```diff
@@ -76,6 +90,17 @@ def get_tx(self):
             'chainId': 111
         }
 
+    def get_tx_2(self):
```

Comment:
Please, no need to create unnecessary entities by copying. Reuse already exists code.

Reply:
Fixed